coderadar-test/src/test/java/io/reflectoring/coderadar/rest/analysis/PaySimProjectIntegrationTest.java

```diff
@@ -38,10 +38,7 @@
 import java.util.List;
 import net.lingala.zip4j.ZipFile;
 import org.apache.tomcat.util.http.fileupload.FileUtils;
-import org.junit.jupiter.api.AfterAll;
-import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.BeforeAll;
-import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.*;
```

Comment:
Haben wir Restriktionen, ab wann der *-Import genutzt werden darf?

Reply:
I've never touched the imports in Coderadar, this is handled by spotless and IntelliJ automatically.